.werft/jobs/build/prepare.ts

```diff
@@ -84,7 +84,7 @@ function decideHarvesterVMCreation(werft: Werft, config: JobConfig) {
     if (shouldCreateVM(config)) {
         createVM(werft, config)
     } else {
-        werft.currentPhaseSpan.setAttribute("werft.harvester.created_vm", false)
+        werft.phases[werft.currentPhase].span.setAttribute("werft.harvester.created_vm", false)
```

Comment:
Can you introduce a method `getCurrentPhaseSpan()`?

Reply:
Done